frontend/test/metabase/scenarios/admin/datamodel/metrics.cy.spec.js

```diff
@@ -263,5 +263,22 @@ describe("scenarios > admin > datamodel > metrics", () => {
       cy.findByText("13022_Metric"); // Name
       cy.findByText("Orders, CE"); // Definition
     });
+
+    it("should show CE that uses 'AND/OR' (metabase#13070)", () => {
+      cy.visit("/admin/datamodel/metrics");
+      cy.findByText("New metric").click();
+      cy.findByText("Select a table").click();
+      cy.findByText("Orders").click();
+      cy.findByText("Add filters to narrow your answer").click();
+      cy.findByText("Custom Expression").click();
+      cy.get(".ace_text-input")
+        .click()
+        .clear()
+        .type("[ID] > 0 OR [ID] < 9876543210", { delay: 100 });
```

Comment:
Not sure we need this workaround anymore. It was valid while we still had to deal with the `contenteditable`.

Reply:
Very true, I've simplified it.